generate_mgit.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
not sure if this is something other people would use and not sure if this is the right place to have it.

Reply:
Seems as good a place as any, and could be useful to others! But maybe put it inside a directory called e.g. `dev-utils`?

Reply:
done